tests/manager/integration/license/opds/for_distributors/test_api.py

```diff
@@ -143,173 +96,6 @@ def test_can_fulfill_without_loan(
         lpdm.delivery_mechanism.drm_scheme = DeliveryMechanism.BEARER_TOKEN
         assert True == m(patron, pool, lpdm)
 
-    def test_get_token_success(
```

Comment:
I did some checking to make sure that removed tests cases here are covered by the tests cases in `TestOAuthOpdsRequest`.